package.json

```diff
@@ -71,6 +74,8 @@
     "istanbul": "0.4.3",
     "mocha": "2.4.5",
     "npm-run-all": "2.1.1",
+    "postcss-cli": "2.6.0",
+    "postcss-import": "8.1.2",
```

Comment:
I added it because I think splitting the CSS into components is a good idea, and this will allow us to do that.